app/src/main/java/com/mapzen/erasermap/AndroidModule.java

```diff
@@ -116,7 +126,7 @@ public AndroidModule(EraserMapApplication application) {
 
             @Override public Map<String, String> queryParamsForRequest() {
                 Map<String, String> params = new HashMap<>();
-                params.put(ApiKeyConstants.API_KEY, apiKeys.getApiKey());
+                params.put(Http.PARAM_API_KEY, apiKeys.getApiKey());
```

Comment:
This is for interaction with Pelias so we need to append the api key. When https://github.com/mapzen/android/pull/412 merges, we can use the new `MapzenSearchHttpRequestHandler` to append only the DNT header (the api key will be handled by the SDK)